transport/src/test/java/io/camunda/zeebe/transport/stream/impl/StreamIntegrationTest.java

```diff
@@ -59,7 +62,10 @@ final class StreamIntegrationTest {
           .setIoBoundActorThreadCount(1)
           .build();
   private final List<Node> clusterNodes =
-      List.of(createNode("server1"), createNode("server2"), createNode("client"));
+      List.of(
+          createNode(UUID.randomUUID().toString()),
+          createNode(UUID.randomUUID().toString()),
+          createNode(UUID.randomUUID().toString()));
```

Comment:
:question: I don't understand why make them random? Does it have any impact?

Reply:
I changed it originally when I was sending the node ID in the header of the message to recreate streams (and couldn't be a string), but it is no longer needed since we now use the method that knows who sent the message.